parsers/sig.py

```diff
@@ -152,12 +156,12 @@ def parse(self, sig_text):
 
     # infer method, dose_unit, and route from NDC or RXCUI
     def infer(self, match_dict, ndc=None, rxcui=None):
-        #sig_elements = ['method', 'dose_unit', 'route']
-        sig_elements = ['method', 'route']
+        sig_elements = ['method', 'dose_unit', 'route']
+        #sig_elements = ['method', 'route']
         inferred = dict.fromkeys(sig_elements)
         for sig_element in sig_elements:
             inferred[sig_element] = infer_sig_element(sig_element, ndc, rxcui)
-        inferred['sig_readable'] = self.get_readable(match_dict, inferred_method=inferred['method'], inferred_route=inferred['route'])
+        inferred['sig_readable'] = self.get_readable(match_dict, inferred_method=inferred['method'], inferred_route=inferred['route'], inferred_form=inferred['dose_unit'])
```

Comment:
Please use `dose_unit` instead of `form` to be consistent with other parts of the app.